js/blocks.js

```diff
@@ -1000,9 +1007,10 @@ function Blocks(canvas, stage, refreshCanvas, trashcan, updateStage, getStageSca
                 }
             }
 
-            // console.log('Adjust Docks: ' + this.blockList[newBlock].name);
+            console.log('Adjust Docks: ' + this.blockList[newBlock].name);
```

Comment:
Do we really need new debug message being sent to console in production release?